dandi/validate.py

```diff
@@ -41,6 +42,8 @@ def validate_file(filepath, schema_version=None, devel_debug=False):
         return validate_dandiset_yaml(
             filepath, schema_version=None, devel_debug=devel_debug
         )
+    elif os.path.splitext(filepath)[-1] in EXTERNAL_FILE_EXTENSIONS:
+        return []
```

Comment:
IMHO we shouldn't just pretend that there are no errors if we are not validating such files.
Since NWB files AFAIK should always have `.nwb` extension by convention, let's just convert `else:` into `if os.path.splitext(filepath)[-1] == '.nwb'`?
even though in effect it might be the same (we return `None` instead of an empty list of errors), at least it would not pretend to validate them. WDYT?